sdk/purview/azure-analytics-purview-workflow/src/test/java/com/azure/analytics/purview/workflow/PurviewWorkflowClientTestBase.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.analytics.purview.workflow;
+
+import com.azure.core.credential.AccessToken;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.TestMode;
+import com.azure.core.util.Configuration;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import java.time.OffsetDateTime;
+
+import com.azure.identity.UsernamePasswordCredentialBuilder;
+import reactor.core.publisher.Mono;
+
+class PurviewWorkflowClientTestBase extends TestBase {
+    protected PurviewWorkflowClient purviewWorkflowClient;
+
+    @Override
+    protected void beforeTest() {
+        PurviewWorkflowClientBuilder purviewWorkflowClientbuilder =
+                new PurviewWorkflowClientBuilder()
+                        .endpoint(Configuration.getGlobalConfiguration().get("ENDPOINT", "endpoint"))
+                        .httpClient(HttpClient.createDefault())
```

Comment:
The code generation should be updated to use the [AssertingClient ](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-test/src/main/java/com/azure/core/test/http/AssertingClient.java) to confirm sync stack changes.

@srnagar Also, now that the sync stack changes are more stable, can we make enable sync stack by default on generation so that newer generations always include those changes for DPG libs. Thoughts?

Reply:
Got it. Let me see if I can change the codegen to use AssertingClient for TestBase.

enable-sync-stack is the default for DPG in later versions.
We also plan to make it default for TSP as well.